content/articles/using-logistic-regression-in-pytorch-to-identify-handwritten-digits/index.md

```diff
@@ -0,0 +1,169 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /implementing-gan-from-scratch/
+title: Implementing GANs from Scratch
+description: 
+author: bonface-ndolo
+date: 2021-11-19T00:00:00-18:00
+topics: [Languages]
+excerpt_separator: <!--more-->
+images:
+
+ - url: /engineering-education/implementing-gan-from-scratch/hero.jpg
+   alt: Implementing GANs example image
+---
+
+### Introduction
+The odds ratio can be calculated using logistic regression when there are multiple explanatory variables. The approach is quite similar to multiple linear regression, except that the response variable is a binomial. An event's odds ratio is the final result of each variable's influence on the odds ratio.
+
+To predict a binary result, we use the statistical technique of logistic regression. The MNIST dataset contains handwritten digits. Therefore we'll use Logistic Regression in PyTorch to figure out which ones they are.
+
+### Prerequisites
+1. Install [PyTorch](https://pytorch.org/) into your Python environment.
+2. Having some knowledge of the Python programming language is a good idea.
+
+### Table of contents
+- [Library functions and objects](#library-functions-and-objects)
+- [Defining our hyperparameters](#defining-our-hyperparameters)
+- [Building The Neural Network](#building-the-neural-network)
+- [References ](#references )
+- [Conclusion](#conclusion)
+ 
+### Library functions and objects
+Importing Library functions and objects into your code is necessary once your installation is complete.
+Here are the functions and objects to import:
+
+- The **torch.nn** module: Contains code for the required model.

```

Comment:
What model are you talking about? Doesn't it contain abstract implementations of a components of a nn?